cmds/dmesg/dmesg.go

```diff
@@ -5,33 +5,52 @@
 // Read the system log.
 //
 // Synopsis:
-//     dmesg [-c]
+//     dmesg [-clear|-read-clear]
 //
 // Options:
-//     -c: clear the log
+//     -clear: clear the log
+//     -read-clear: clear the log after printing
 package main
 
 import (
 	"flag"
 	"log"
 	"os"
+	"syscall"
 	"unsafe"
+)
 
-	"golang.org/x/sys/unix"
+var (
+	clear     = flag.Bool("clear", false, "Clear the log")
+	readClear = flag.Bool("read-clear", false, "Clear the log after printing")
```

Comment:
we should preserve -c, it's just way more convenient ...

Reply:
Done.